cmd/gpu_plugin/labels.md

```diff
@@ -0,0 +1,85 @@
+# Labels
+
+GPU labels originate from two main sources: NFD rules and GPU plugin (& NFD hook).
+
+## NFD rules
+
+NFD rule is a method to instruct NFD to add certain label(s) to node based on the devices detected on it. There is a generic rule to identify all Intel GPUs. It will add labels for each pci device type. For example, a Tigerlake iGPU will show up as:
```

Comment:
There can be other TigerLake variants:
```suggestion
NFD rule is a method to instruct NFD to add certain label(s) to node based on the devices detected on it. There is a generic rule to identify all Intel GPUs. It will add labels for each PCI device type. For example, a Tigerlake iGPU (with PCI ID 0x9a49) will show up as:
```